core/templates/filters/string-utility-filters/normalize-whitespace.pipe.ts

```diff
@@ -27,7 +27,7 @@ import { UtilsService } from 'services/utils.service';
 @Pipe({name: 'normalizeWhitespace'})
 export class NormalizeWhitespacePipe implements PipeTransform {
   constructor(private utilsService: UtilsService) {}
-  transform(input: any): any {
+  transform(input: string): string {
     if (this.utilsService.isString(input)) {
```

Comment:
Do we still need this check? It should be checked by the TypeScript right?

Reply:
I think we need this if we get some invalid input in runtime like null, undefined than the function should just return that invalid value.